public/javascript/getHelp.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file getHelp.js
+ * Handler for showing functions
+ *
+ */
+ 
+ module.exports = function setThreadColor(api, threadID, body) {
+        function sleep(milliseconds) {                  //For message timing
+                var start = new Date().getTime();
+                for (var i = 0; i < 1e7; i++) {
+                        if ((new Date().getTime() - start) > milliseconds){
+                                break;
+                        }
+                }
+        }
+        
+        var worked = 1;
+        var command = body.substring('@help'.length);
+        if(command.length==0) {
+                api.sendMessage("Note: Add @ at the beginning of every command", threadID);
+                sleep(300);
```

Comment:
What is the point of sleep here? Since the bot is not async, this will hung up the bot during a huge load of requests.